### ISSUE TITLE
FEATURE - časový limit príchod / odchod

### ISSUE THREAD

**Issue by winky1404** [psekan/somvrade#30]
Keďže kultivácia Covid-19 vzorky predstavuje 15-20 minút po výtere, navrhujem, aby odstup medzi časmi bol minimálne 15 minút, aby sa predišlo tomu, aby appka accidentally napisala čas príchodu a odchodu rovnaký, alebo aby ludia neklikali hore dole. Keď už, nech to má nejakú hodnotu ten údaj. Čiže človek Ak sa zahlási o 15:00, nech mu nedovolí nahodiť čas odchodu skôr ako +15 minút, čiže najskorší čas odchodu môže byť v tomto prípade 15:15. Rovnako, testovanie začína o 07:00, a niekto sa nahlási na čakanie o 6:30, nech najskorší čas odchodu je možný od 07:15.

**Comment by WW0nka** [comment]
Nadvazujem na moj koment k #35 a sice, nie kazdy v rade ostava, po zavedeni casovych limitov nevieme pri spracovani casov rozlisit medzi 1. clovek odisiel z radu a 2. osobu velmi rychlo otestovali.

**Comment by winky1404** [comment]
áno, daný limit by sa mal aplikovať iba tam, kde úspešný "uchádzač" došiel až po stage že "som hotový", a ak by sa len tak preklikal z nudy od body jedna až sem, jednoducho by ho to nepustilo, lebo by to bol fake a kiddo, čo jednoducho postlo fake udaje, preto ten limit ... a aj keby osobu velmi rychlo otestovali, 15 minut tam je minimum ... čisa mýlim ? :)

**Comment by WW0nka** [comment]
nemylis len narazam na to, co riesime v #34 a ludi typu "observer"

**Comment by winky1404** [comment]
po zapracovaní myšlienky observer, ktorý by mal iné možnosti, tak pre bežneho smrtelnika je interval skutočne anti-dement riešenie . najprv teda asi vyriešiť observerov, potom sa môžeme baviť o tom limite. to už ale necham na vyvojarov, ako sas tym popasuju, a tipnem že ty si jednym z nich :)
